tests/framework/installer/nfs_manifests.go

```diff
@@ -43,23 +43,24 @@ spec:
 
 // GetNFSServerOperator returns the NFSServer operator definition
 func (i *NFSManifests) GetNFSServerOperator(namespace string) string {
-	return `kind: Namespace
-apiVersion: v1
+	return `apiVersion: v1
+kind: Namespace
 metadata:
-  name: ` + namespace + `
+  name:  ` + namespace + `
 ---
 apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: ClusterRole
 metadata:
   name: rook-nfs-operator
 rules:
 - apiGroups:
-  - ""
+  - "*"
```

Comment:
same comment as before about using the explicit api group instead of a wildcard

Reply:
changed